website/views/layouts/layout.ejs

```diff
@@ -86,19 +86,21 @@
-              <a href="/blog" class="header-link d-flex align-items-center px-3 py-2 mb-4 text-decoration-none" style=" text-decoration: none; color: #192147; font-weight: 700;">Blog</a>
-              <!-- <a href="/company/contact" class="header-link d-flex align-items-center px-3 py-2 mb-4 text-decoration-none" style=" text-decoration: none; color: #192147; font-weight: 700;">Contact</a> -->
-              <a target="_blank" href="https://github.com/fleetdm/fleet" class="header-link d-flex align-items-center px-3 py-2" style=" text-decoration: none; color: #192147; font-weight: 700;">
-                <img alt="A blue octocat icon representing the GitHub brand." src="/images/github-mark-blue-24x24@2x.png" style="width: 24px; font-weight: 700;" />
+              <a href="/pricing" class="menu-link d-flex align-items-center px-3 py-2 mb-4 text-decoration-none" style=" text-decoration: none; font-weight: 700;">Pricing</a>
+              <a href="/blog" class="menu-link d-flex align-items-center px-3 py-2 mb-4 text-decoration-none" style=" text-decoration: none; font-weight: 700;">Blog</a>
+              <!-- <a href="/company/contact" class="menu-link d-flex align-items-center px-3 py-2 mb-4 text-decoration-none" style=" text-decoration: none; color: #192147; font-weight: 700;">Contact</a> -->
+              <a target="_blank" href="https://github.com/fleetdm/fleet" class="menu-link d-flex align-items-center px-3 py-2" style=" text-decoration: none; font-weight: 700;">
+                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M11.9633 0.5C5.3578 0.5 0 5.8578 0 12.4633C0 17.7477 3.44954 22.2248 8.14679 23.8394C8.73394 23.9128 8.95413 23.5459 8.95413 23.2523C8.95413 22.9587 8.95413 22.2248 8.95413 21.1972C5.65138 21.9312 4.91743 19.5826 4.91743 19.5826C4.40367 18.1881 3.59633 17.8211 3.59633 17.8211C2.49541 17.0872 3.66972 17.0872 3.66972 17.0872C4.84404 17.1606 5.50459 18.3349 5.50459 18.3349C6.6055 20.1697 8.29358 19.656 8.95413 19.3624C9.02752 18.555 9.3945 18.0413 9.68807 17.7477C7.04587 17.4541 4.25688 16.4266 4.25688 11.8028C4.25688 10.4817 4.69725 9.45413 5.50459 8.57339C5.43119 8.35321 4.99083 7.1055 5.65138 5.49083C5.65138 5.49083 6.6789 5.19725 8.95413 6.73853C9.90826 6.44495 10.9358 6.37156 11.9633 6.37156C12.9908 6.37156 14.0183 6.51835 14.9725 6.73853C17.2477 5.19725 18.2752 5.49083 18.2752 5.49083C18.9358 7.1055 18.4954 8.35321 18.422 8.64679C19.156 9.45413 19.6697 10.555 19.6697 11.8761C19.6697 16.5 16.8807 17.4541 14.2385 17.7477C14.6789 18.1147 15.0459 18.8486 15.0459 19.9495C15.0459 21.5642 15.0459 22.8119 15.0459 23.2523C15.0459 23.5459 15.2661 23.9128 15.8532 23.8394C20.6239 22.2248 24 17.7477 24 12.4633C23.9266 5.8578 18.5688 0.5 11.9633 0.5Z" fill="#192147"/>
+                </svg>
```

Comment:
@gillespi314 I think it's worth checking https://caniuse.com/svg versus our supported browsers in layout.ejs.  If all good there, then this is ok with me -- though note there's no `alt` text so it's less accessible for people who use screen readers vs. an <img> tag.  And actually, we also will probably hook up retina.js or equivalent at some point.  So `<img>` tag would be best.  See other images for our naming convention- the pixels in the filename are always 1/2 the real pixel size because they're CSS pixels (this is so images show nicely on retina displays)

Reply:
Looks like svg is broadly supported although there are some issues with proper scaling in IE and Edge that can be addressed with the width, height, viewBox and CSS rules as a workaround.  

The svg approach for the github logo was used in another part of the website so I followed it here.  If you think it makes sense to move to `<img>` instead, I can find a version of the logo that we can style to match our color scheme and change it in the other places the svg appears.

Reply:
> I can find a version of the logo that we can style to match our color scheme and change it in the other places the svg appears.

I think this is worthwhile, thank you